make.ps1

```diff
@@ -4,7 +4,7 @@ Param(
     [Parameter(Position=1)]
     [String] $target = "release",
     [String] $configuration = "Release",
-    [String[]] $frameworks=@('net45','netcoreapp2.0'),
+    [String[]] $frameworks=@('net45','netcoreapp2.1'),
```

Comment:
You don't want to build netcoreapp2.0 anymore?

Reply:
The list of frameworks is only used for testing. We'd probably run out of time if we tested both 2.0 and 2.1.

Reply:
Ah, right.